docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: "Set Up"
+sidebar_order: 10
+description: "Get started with Sentry's Dev Toolbar, bringing critical Sentry insights and tools into your web app to help your team troubleshoot more effectively.."
+---
+
+<Alert level="info">
+  The Dev Toolbar is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
+</Alert>
+
+## Set Up Your Web App
+
+<Alert title="Note" level="warning">
+    [Enabling tracing](/platforms/javascript/tracing/) is a prerequisite for having page-aware panels, as it allows us to collect page-specific issues and feedback through transactions.
+</Alert>
```

Comment:
Are you saying that you _could_ use the Dev Toolbar without tracing? Or that if you have tracing you will get page-aware panels but if you don't have tracing you will get the issues across that project but not necessarily for that page?

Regardless, I think this should be clarified depending on your answer.